src/main/java/com/google/sps/user/User.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.user;
+
+import org.jetbrains.annotations.Nullable;
+
+/** Stores the data related to one user. */
+public class User {
+    private final String id, email;
+    private final boolean publicPortfolio;
```

Comment:
I thought we agree line indentation is 2 spaces?

Reply:
Yes, sorry, I didn't find this setting in the cloud shell editor. 
I switched to intellij now, and changed the indentation.